sympy/integrals/manualintegrate.py

```diff
@@ -1553,9 +1593,35 @@ def eval_reciprocal_sqrt_quadratic(a, b, c, integrand, x):
     return log(2*sqrt(c)*sqrt(a+b*x+c*x**2)+b+2*c*x)/sqrt(c)
 
 
+@evaluates(SqrtQuadraticDenomRule)
+def eval_sqrt_quadratic_denom(a, b, c, coeffs: list[Expr], integrand, x):
+    # Integrate poly/sqrt(a+b*x+c*x**2) using recursion.
+    # coeffs are coefficients of the polynomial.
+    # See https://github.com/sympy/sympy/pull/23608 for formula.
```

Comment:
Put a reference here as it seems a common way
```sh
$ git grep -n 'https://github.com/sympy/sympy/pull' **/*.py
```

Reply:
I think it's fine to have the formula here in the comment. Just the formula shouldn't be in LaTex format because that's not the easiest to read (and the point of the comments is to be read when looking at the code).

Reply:
Now it should be fine.